test/components/plots/plotTests.ts

```diff
@@ -34,24 +34,24 @@ describe("Plots", () => {
       var r = new CountingPlot();
       r.addDataset("foo", dFoo);
 
-      assert.equal(1, r.renders, "initial render due to addDataset");
+      assert.strictEqual(1, r.renders, "initial render due to addDataset");
```

Comment:
Not sure if part of this commit, but as a TODO, this is inconsistent. First value is the given, second is the expected (at least in the rest of the testing codebase)